File: 1758/solution.go
package main

func minOperations(s string) int {
	res1 := 0
	res2 := 0
	for i, char := range s {
		if int(char)-'0' == i%2 {
			res2++
		} else {
			res1++
		}
	}
	if res2 > res1 {
		return res1
	} else {
		return res2
	}
}

File: 1758/solution.java
class Solution {
    public int minOperations(String s) {
        int res1 = 0;
        int res2 = 0;
        for (int i = 0; i < s.length(); i++) {
            if (i % 2 == s.charAt(i) - '0') {
                res1++;
            } else {
                res2++;
            }
        }
        return Math.min(res1, res2);
    }
}